### PR TITLE
Migrate gke-1.5/1.6 jobs from jenkins to prow

### PR DESCRIPTION
**Pull request by krzyzacy**
Hit by https://github.com/kubernetes/release/blob/master/lib/releaselib.sh#L106, when making a 1.7 release, so I'd better make everything consistent here asap.

cc @rmmh this renames stuff from 1.5/1.6 to 1-5/1-6 so we don't need to worry about the build number here.

/assign @caesarxuchao  @fejta

### CODE REVIEW

File: experiment/jenkins_to_prow.py [hunk 1]
```diff
@@ -27,7 +27,7 @@
 
 TEMPLATE = {
     'name': '',
-    'interval': '6h',
```
Comment (spiffxp) [inline experiment/jenkins_to_prow.py]: looks like 1.5 jobs were run with this at 6h, and 1.6 jobs at 2h, was that the intent?

Comment (krzyzacy) [inline experiment/jenkins_to_prow.py]: yes, older jobs runs less often.

**Comment by krzyzacy** [comment]
/assign @ixdy 
gonna rebase

**Comment by rmmh** [comment]
I would rather we keep the names the same so history isn't erased.

**Comment by krzyzacy** [comment]
@rmmh I'd rather too, the issue is k8s dns does not like dots in the name.

**Comment by ixdy** [comment]
one advantage of renaming is that we don't have to figure out how not to clobber build numbers in GCS.

**Comment by rmmh** [comment]
Why could the jobs have dots in the name before and not now?

Not clobbering build numbers in GCS is not hard, it needs a minor adjustment to tot.

**Comment by krzyzacy** [comment]
@rmmh because now we are in prow, and the real problem is here - https://github.com/kubernetes/kubernetes/blob/5b3d0bbe66343e14eade6b1c746f868aee1a6fa9/staging/src/k8s.io/apimachinery/pkg/util/validation/validation.go#L92

**Comment by krzyzacy** [comment]
@rmmh I was trying to do something like #3000 but seems people have different ideas. I think we'd need some agreement on how to move forward :-)

**Comment by fejta** [comment]
/lgtm

Preserving history is IMO not important. Testgrid only shows a week of history anyway.

**Comment by krzyzacy** [comment]
bumping config, and I'll write the prow plugin.

**Pull request merged**